modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/shared/ClayButton.es.js

```diff
@@ -0,0 +1,79 @@
+import React, {Component} from 'react';
+import {PropTypes} from 'prop-types';
+import ClayIcon from './ClayIcon.es';
+import getCN from 'classnames';
+
+class ClayButton extends Component {
+	static propTypes = {
+		borderless: PropTypes.bool,
+		className: PropTypes.string,
+		iconName: PropTypes.string,
+		label: PropTypes.string,
+		monospaced: PropTypes.bool,
+		style: PropTypes.oneOf(
+			[
+				'primary',
+				'secondary',
+				'info',
+				'success',
+				'warning',
+				'danger',
+				'dark',
+				'light',
+				'unstyled'
+			]
+		),
+		type: PropTypes.string
+	};
+
+	static defaultProps = {
+		borderless: false,
+		monospaced: false,
+		style: 'secondary',
+		type: 'button'
+	};
+
+	render() {
+		const {
+			borderless,
+			className,
+			iconName,
+			label,
+			monospaced,
+			style,
+			type,
+			...otherProps
+		} = this.props;
+
+		const stylePrefix = borderless ? 'btn-outline-' : 'btn-';
+
+		const classes = getCN(
+			'btn',
+			`${stylePrefix}${style}`,
+			{
+				'btn-monospaced': monospaced,
+				'btn-outline-borderless': borderless
+			},
+			className
+		);
+
+		return (
+			<button
+				className={classes}
+				type={type}
+				{...otherProps}
+			>
+				{label}
+
+				{iconName && (
+					<ClayIcon
+						className={`${label ? 'text-icon' : 'icon'}`}
```

Comment:
Since there is no other string outside the condition, you don't need to wrap it in a template-literal.